assets/src/scripts/components/hydrograph/index.spec.js

```diff
@@ -163,9 +163,12 @@ describe('Hydrograph charting module', () => {
         component.append('div').attr('class', 'provisional-data-alert');
 
         graphNode = document.getElementById('hydrograph');
+
+        jasmine.Ajax.install();
```

Comment:
We were actually making NWIS service calls as part of the test suite. This may be one reason our travis builds sometimes failed. By adding this, all ajax calls are stubbed.

Reply:
Nice find